uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/fields-templates/leave-type-wizard-field-category.html

```diff
@@ -1,13 +1,13 @@
 <div class="row form-horizontal">
   <div class="col-md-2"
     ng-repeat="leaveTypeCategory in form.leaveTypeCategories">
-    <div class="well pointer text-center" ng-click="field.value = leaveTypeCategory.value">
+    <div class="well pointer text-center" ng-click="field.value = leaveTypeCategory.name">
```

Comment:
Because in this PR we load the categories from the option group and their `name` is what needs to be passed to API. Previously, we simply had a had a hardcoded collection of categories, where I used `value` property for their names.